lib/person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] As requested by the previous reviewer and according to the project requirements, this method is supposed to be named as is_of_age and not of_age. Kindly fix this.

![image](https://user-images.githubusercontent.com/86318284/200564871-459f3c0a-0b67-4a02-8a4f-adef7d56b06b.png)

Reply:
Hello @Nemwel-Boniface thank you for your feedback. I changed it because of the `Rubocop` recommendation. According to the Ruby style guild, only a predefined method should start `is`. [Please check here](https://rubystyle.guide/#bool-methods-prefix).

Reply:
Hello @melashu I have reconfirmed this. Kindly request for a review in your dashboard and it should be approved.

Reply:
Thank you, Sir!